sync2jira/downstream_issue.py

```diff
@@ -81,12 +87,15 @@ def get_snowflake_conn():
     )
 
 
-def execute_snowflake_query(title, issue):
+def execute_snowflake_query(issue):
+    if not validate_github_url(issue.url):
+        log.error(f"Invalid GitHub URL format: {issue.url}")
+        return []
     conn = get_snowflake_conn()
     # Execute the Snowflake query
     with conn as c:
         cursor = c.cursor()
-        cursor.execute(SNOWFLAKE_QUERY.format(title, issue.url))
+        cursor.execute(SNOWFLAKE_QUERY, (issue.url,))
```

Comment:
I see you're taking advantage of the Snowflake client's [client-side variable binding](https://docs.snowflake.com/en/developer-guide/python-connector/python-connector-example#pyformat-or-format-binding).  Did you see [their recommendation](https://docs.snowflake.com/en/developer-guide/python-connector/python-connector-example#avoid-sql-injection-attacks) to use server-side binding to avoid SQL injection?